test/src/test/java/org/corfudb/integration/CorfuReplicationClusterConfigIT.java

```diff
@@ -1007,6 +1011,140 @@ public void testActiveLockRelease() throws Exception {
         assertThat(mapStandby.size()).isEqualTo(secondBatch);
     }
 
+    /**
+     * This test verifies log entry works after a force snap sync
+     * in the backup/restore workflow.
+     * <p>
+     * 1. Init with corfu 9000 active, 9001 standby, 9002 backup
+     * 2. Write 10 entries to active map
+     * 3. Start log replication: Node 9010 - active, Node 9020 - standby, Node 9030 - backup
+     * 4. Wait for Snapshot Sync, both maps have size 10
+     * 5. Write 5 more entries to active map, to verify Log Entry Sync
+     * 6. Change the topology, backup 9030 will be the new active
```

Comment:
I think these descriptions are nice, but here we are missing some steps as it is not clear that backup lags behind, maybe we can add this info so it fully explains the test.

Reply:
yes, added a few more description about the backup